test/dialect/mysql/test_compiler.py

```diff
@@ -558,6 +558,50 @@ def test_column_computed(self, text, persisted):
             "ALWAYS AS (x + 2)%s)" % text,
         )
 
+    @testing.combinations(
```

Comment:
**mike bayer** (zzzeek) wrote:

I would like to have some round trip tests for this added, basically this exact same test but doing a round trip.  that way we have the assert_raises checking for the database server error as we are removing the CompileError part of this.    This new test can be in mysql/test_query.py and I would make a new suite ComputedTest.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4869

Reply:
Updated as per comments, moved test to test_query.py and droped explicit compile error